internal/otelcollector/config/metric/agent/testdata/config_compatibility_enabled.yaml

```diff
@@ -0,0 +1,283 @@
+extensions:
```

Comment:
Please remove this test data file if it is not used anymore in any of the unit tests